File: Number of Unequal Triplets in Array.py
class Solution(object):
    def unequalTriplets(self, nums):
        n=len(nums)
        c=0
        #Checks all triplets
        for i in range(n):
            for j in range(i+1, n):
                for k in range (j+1, n ):
                    if nums[i]!=nums[j] and nums[i]!=nums[k]and nums[j]!=nums[k]: #cheack each nomber in the array and thenumber of occurrences on each to avoid repetition
                        c+=1 #only adds one if the number in the array has not yet been passed and counted
        return c

File: Number of Unequal Triplets in Array.java
class Solution {
    public int unequalTriplets(int[] nums) {
        int n=nums.length;//number to iterate 
        int c=0; //counter inicialization
        for(int i =0;i<n;i++){
            for(int j =i+1;j<n;j++){
                for(int k=j+1;k<n;k++){
                    if(nums[i]!=nums[j]&&nums[i]!=nums[k]&&nums[j]!=nums[k]){//checks if the number on each position of the array is equal to the other, 
                        c++;//adds to the couter the number of times a symbol does not repeat in the other positions of the array
                    }
                }
            }
        }
     return c;

    }
}
